README.md

```diff
@@ -1,13 +1,192 @@
-# appsflyer-framework-ars-beta
-## Adding SDK to your project 
+<img src="https://massets.appsflyer.com/wp-content/uploads/2018/06/20092440/static-ziv_1TP.png"  width="400" > 
 
-To install the SDK using `CocoaPods`, add  `'pod 'PurchaseConnector'` to your Podfile and run `pod update`.
+# iOS Purchase Connector
 
-Note: Please make sure to install and set up AppsFlyerFramework pod version `6.6.0` or later `pod 'AppsFlyerFramework'`
+[![License: MIT](https://img.shields.io/badge/License-MIT-blue.svg)](https://github.com/AppsFlyerSDK/android-purchase-connector/blob/main/LICENSE)
 
-## ARS SDK Initialization - Swift Example 
+🛠 In order for us to provide optimal support, we would kindly ask you to submit any issues to
+support@appsflyer.com
 
+> *When submitting an issue please specify your AppsFlyer sign-up (account) email , your app ID , production steps, logs, code snippets and any additional relevant information.* 
+
+## Table Of Content
+  * [This Module is Built for](#plugin-build-for)
+  * [Adding The Connector To Your Project via Cocoapods](#install-connector)
+  * [Basic Integration Of The Connector](#basic-integration)
+    + [Set up Purchase Connector](#create-instance)
+    + [Log Auto-Renewable Subscriptions and In-App Purchases](#log-subscriptions)
+    + [Conform to Purchase Connector Data Source and Delegate protocols](#conforming)
+    + [Start Observing Transactions](#start)
+    + [Stop Observing Transactions](#stop)
+  * [Testing the implementation in Sandbox](#testing)
+  * [Full Code Examples](#example)
+
+
+## <a id="plugin-build-for"> This Module is Built for
+- iOS version 9 and higher.
+- iOS AppsFlyer SDK **6.6.1** and higher.
```

Comment:
I believe we added support for in app purchase only in version 6.8.0, no?

Reply:
@guysarfatty-af in-app purchase feature is not dependent on the SDK version, as far as I know. The minimum iOS version for the Purchase connector is 6.6.1, in future changes if it changes, we will update the documentation.